app/templates/pdf/attendees_pdf.html

```diff
@@ -107,34 +107,33 @@ <h3 style="text-align:center;">{{ holders[0].event.name }} - {{ ("Attendees List
 
                 </td>
                 <td style="text-align:center;"><br>
-                    <strong>
-                        First Name :<br>
-                        Last Name :<br>
-                        Email :<br>
-                    </strong>
+                    {% for field in custom_forms %}
+                        {{ field.name }} <br>
+                     {% endfor %}
                     Amount :<br>
                     Ticket Name :<br>
                     Status :<br>
-                </td>
-                <td><br>
-                    <strong>
-                        {{ holder.firstname }}<br>
-                        {{ holder.lastname }}<br>
-                        {{ holder.email }}<br>
-                    </strong>
+                 </td>
+                 <td><br>
+                     {% for field in custom_forms %}
+                        {% if field.is_complex %}-->
+                            {{ holder.complex_field_values | attr(field.field_identifier) }} {{ holder.complex_field_values | attr(field.field_identifier) }} <br>
```

Comment:
@iamareebjamal - Need your help here, how do I apply a condition when the attribute is not present.

Reply:
if field.is_complex and holder.complex_field_values

Reply:
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/20026440/99094174-7847fc00-25f9-11eb-9ea4-16549a270a2a.png">

Like an arrows are developing here I want this to be `''`

Reply:
Remove `-->`

Reply:
O my eyes 👀

Reply:
@iamareebjamal - Still the values are not fetched by ` holder.complex_field_values | attr(field.field_identifier)` will check this

Reply:
Fixed